index.html

```diff
@@ -55,5 +55,162 @@ <h2 class="header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+    <section class="recent-container" id="porto">
+      <div class="work-container">
+        <h2 class="recent-work">My Recent Works</h2>
+        <hr>
+      </div>
+      <div class="image-p">
```

Comment:
- [ ] please ensure you are using the same font style, weight, font family and font size like the figma version
- [ ] kindly add the sizes of your tech buttons to be more visible
- [ ] kindly ensure your card headings are bolder
- [ ] please implement this changes for all the works cards.
your version

![Screenshot (306)_LI](https://user-images.githubusercontent.com/65298048/185484782-9626275b-c49f-4865-a138-5a539c112010.jpg)

figma  version
![Screenshot (305)_LI](https://user-images.githubusercontent.com/65298048/185484677-17344044-d418-48ab-82cb-d7998ca9a8ef.jpg)